modules/ROOT/pages/tips-and-tricks.adoc

```diff
@@ -0,0 +1,119 @@
+= Tips and Tricks
+
+== Working with Toolbx
+
+=== Running applications from inside Toolbx on the host
+
+This can be necessary if you want to interact with tools available from the
+host, for example `podman`, `nmcli` or `rpm-ostree` without leaving the Toolbx
+container in between. You can use `flatpak-spawn`, included in the base
+installation for this:
+
+  $ flatpak-spawn --host podman --help
+
+If the application you want to call requires `sudo` access, the `-S` option must
+be supplied to `sudo` like below:
+
+  $ flatpak-spawn --host sudo -S rpm-ostree status
+
+If you find yourself using commands like these frequently to access e.g. the
+flatpak command from inside the Toolbx container, you can create yourself a
+short custom wrapper script (*inside the Toolbx container*):
+
+  $ echo '#!/bin/sh
+    exec /usr/bin/flatpak-spawn --host flatpak "$@"' | \
+    sudo tee /usr/bin/flatpak && sudo chmod +x /usr/bin/flatpak
```

Comment:
Looks like there is something missing here. A redirect to a file maybe?

Reply:
That's what `tee` is there for. I can't just `>` to a file, because I need to be root to write to `/usr/bin`. If I `sudo echo`, the command after the pipe will still be spawned as user process. Alternatively I could also run the whole thing as root and prefix the prompt with `#`, but I find this nicer.

This is also the method you'd usually use to fiddle with e.g. sysfs entries when not working as root all the time.

Reply:
But thinking about it, should I add a note that when the user, after executing this command, sees the following reply: `tee: /usr/bin/foobar: Read-only file system` they forgot to enter their toolbx container? It's Silverblue keeping them from shredding their hosts `flatpak` executable. :)

Reply:
Oh, I had missed the end quote and miss-read the whole thing.
Let's make it one line with `echo -e '...\n...'` and write to `/usr/local/bin` instead?
To make it clear that this has to be run inside a toolbox, you may use `[toolbox]$` for the prompt maybe?